addons/web_editor/models/ir_attachment.py

```diff
@@ -47,4 +47,4 @@ def _compute_image_size(self):
     def _get_media_info(self):
         """Return a dict with the values that we need on the media dialog."""
         self.ensure_one()
-        return self.read(['id', 'name', 'mimetype', 'checksum', 'url', 'type', 'res_id', 'res_model', 'public', 'access_token', 'image_src', 'image_width', 'image_height'])[0]
+        return self.read(['id', 'name', 'description', 'mimetype', 'checksum', 'url', 'type', 'res_id', 'res_model', 'public', 'access_token', 'image_src', 'image_width', 'image_height'])[0]
```

Comment:
don't use read here.
You already have the info in cache

Reply:
just `return self` then?

Reply:
you can use self._read_format au lieu de self.readr.

It has been explicitly created for this use case. (in master)

Same return than read, but using the cache done by previous browse